src/pages/iou/request/step/IOURequestStepTaxAmountPage.tsx

```diff
@@ -34,13 +34,13 @@ type IOURequestStepTaxAmountPageProps = IOURequestStepTaxAmountPageOnyxProps &
         transaction: OnyxEntry<Transaction>;
     };
 
-function getTaxAmount(transaction: OnyxEntry<Transaction>, policy: OnyxEntry<Policy>, isEditing: boolean): number | undefined {
+function getTaxAmount(transaction: OnyxEntry<Transaction>, policy: OnyxEntry<Policy>, currency: string | undefined, isEditing: boolean): number | undefined {
```

Comment:
Can you pass currency from all function usages also, please

Reply:
Passing currency from src/pages/iou/request/step/IOURequestStepAmount.tsx might fix https://github.com/Expensify/App/issues/42049